src/osdp_cp.c

```diff
@@ -1479,6 +1494,14 @@ osdp_t *osdp_cp_setup(int num_pd, const osdp_pd_info_t *info_list)
 	return (osdp_t *)__cp_setup(num_pd, info_list);
 }
 
+int osdp_cp_add_pd(osdp_t *ctx, int num_pd, const osdp_pd_info_t *info)
+{
+	assert(ctx);
+	assert(info);
+
+	return cp_add_pd(ctx, num_pd, info);
```

Comment:
There should be a log line (very similar to the one in line 1476) after the new PD addition. Otherwise, when users post logs with issues, we won't get the full picture.

Reply:
Please, check the last version to see if the msg added is enough.